osv/ecosystems/bioconductor.py

```diff
@@ -59,7 +59,7 @@ def _enumerate_versions(self,
     versions = []
     # Currently breaking on 3.19,
     # see https://github.com/google/osv.dev/pull/1477/files#r1575458933
-    bioc_versions = bioc_versions[1:]
+    bioc_versions = bioc_versions.remove('3.19')
```

Comment:
can we wrap this in a try except (catching ValueError per https://docs.python.org/3/tutorial/datastructures.html#more-on-lists) in case it's not there?

Reply:
Done in [1b1f6f3](https://github.com/google/osv.dev/pull/2130/commits/1b1f6f31e031c24f9ed61baa66b766cc25f48ce5) + [4728b50](https://github.com/google/osv.dev/pull/2130/commits/4728b5044faf129ffd24a985c74cdf31040a3dde)